kimchi/src/verifier.rs

```diff
@@ -43,7 +43,7 @@ where
     /// the computed powers of alpha
     pub all_alphas: Alphas<Fr<G>>,
     /// public polynomial evaluations
-    pub p_eval: [Vec<Fr<G>>; 2],
+    pub p_eval: Vec<Vec<Fr<G>>>,
```

Comment:
What motivates this relaxation of types?

Reply:
that was a fast review o_O basically we later use the polycommitment verification API with vectors of vectors of evaluations. I'm guessing this is because the API is compatible with any number of evaluations (not just 2)